BookingsApi/BookingsApi.DAL/Migrations/20230706110309_AddIsScottishAndIsWorkAllocationEnabledColumn.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using BookingsApi.Domain;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace BookingsApi.DAL.Migrations
+{
+    public partial class AddIsScottishAndIsWorkAllocationEnabledColumn : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<bool>(
+                name: "IsScottish",
+                table: "HearingVenue",
+                type: "bit",
+                nullable: false,
+                defaultValue: false);
+
+            migrationBuilder.AddColumn<bool>(
+                name: "IsWorkAllocationEnabled",
+                table: "HearingVenue",
+                type: "bit",
+                nullable: false,
+                defaultValue: true);
+
+
+            migrationBuilder.UpdateData(table: nameof(HearingVenue),
```

Comment:
we should probably have statements in the `Down()` method to revert these updates

Reply:
ah of course!

Reply:
yep, you're right :)